static/js/tools/stat_var/util.ts

```diff
@@ -34,6 +34,21 @@ export function getUrlToken(param: string): string {
   return urlParams.get(param);
 }
 
+/**
+ * Returns token for URL param, or the given default.
+ * @param param URL param
+ * @def def default value
+ * @returns Corresponding token
+ */
+export function getUrlTokenOrDefault(param: string, def: string): string {
```

Comment:
this file should be util functions specific for stat var explorer .. Since it looks like we're using these util functions across different apps, can we move these functions into a new url_utils.ts file and put that under static/js/utils?

Reply:
Done.